audit.json

```diff
@@ -112,5 +112,6 @@
   "10096_Timestamp Disclosure - Unix_http://wa-task-management-api-aat.service.core-compute-aat.internal/task/task-id_POST": "ignore",
   "10096_Timestamp Disclosure - Unix_http://wa-task-management-api-aat.service.core-compute-aat.internal/task-configuration/task-id/configuration_POST": "ignore",
   "10096_Timestamp Disclosure - Unix_http://wa-task-management-api-aat.service.core-compute-aat.internal/task/task-id/unclaim_POST": "ignore",
-  "10096_Timestamp Disclosure - Unix_http://wa-task-management-api-aat.service.core-compute-aat.internal/task/task-id/notes_POST": "ignore"
+  "10096_Timestamp Disclosure - Unix_http://wa-task-management-api-aat.service.core-compute-aat.internal/task/task-id/notes_POST": "ignore",
+  "100000_A Client Error response code was returned by the server_http://wa-task-management-api-aat.service.core-compute-aat.internal/v2/api-docs_GET": "ignore"
```

Comment:
With the Spring boot upgrade we should be going to `v3/api-docs`

Reply:
this isn't the fix. I have raised this https://github.com/hmcts/wa-task-management-api/pull/535